support-models/src/main/scala/com/gu/support/workers/PaymentProvider.scala

```diff
@@ -0,0 +1,63 @@
+package com.gu.support.workers
+
+import com.gu.support.redemptions.RedemptionData
+import io.circe.{Decoder, Encoder}
+
+sealed trait PaymentProvider
+
+case object Stripe extends PaymentProvider
+
+case object StripeApplePay extends PaymentProvider
+
+case object PayPal extends PaymentProvider
+
+case object DirectDebit extends PaymentProvider
+
+case object Existing extends PaymentProvider
+
+case object RedemptionNoProvider extends PaymentProvider
+
+object PaymentProvider {
+
+  val all = List(
+    Stripe,
+    StripeApplePay,
+    PayPal,
+    DirectDebit,
+    Existing,
+    RedemptionNoProvider
```

Comment:
moved from support-workers project - only this variable was updated as it used to be 
```scala
List(Stripe, PayPal, DirectDebit, Existing)
```